README.md

```diff
@@ -30,12 +30,40 @@ const config = {
         cssAttributes: { rel: 'preload' },
         // Optional
         jsAttributes: { defer: 'defer' }
-      }
+      },
+      // Optional, use this for choosing chunks to include to your page.
+      // See the expanded example below.
+      chunks: ['app']
     })
   ]
 };
 ```
 
+### Multiple pages
+
+In the example below, we generate two separate pages and choose bundles from specific entries to them. This is useful for multi-page apps or sites.
```

Comment:
Can we reword the first sentence to make it more clear? The "choose bundles from specific entries to them" piece doesn't make sense to me.

Reply:
Yeah, I'll think of something.